server/src/main/java/org/apache/druid/segment/metadata/CoordinatorSegmentMetadataCache.java

```diff
@@ -175,7 +175,7 @@ public void stop()
 
   public void leaderStart()
   {
-    log.info("%s starting cache initialization.", getClass().getSimpleName());
+    log.info("Initializing cache.");
```

Comment:
I would recommend adding the name of the class so I can search 
cache %s in the logs :). It makes it easier for me to search thought tons of logs. since each loggers can have its own format. 
Nit: can be done in a followup.